style.css

```diff
@@ -0,0 +1,990 @@
+  .section-main__subtitle { margin-bottom: 89px; }
+
+  .section-main__img { margin-bottom: 44px; }
+
+  .car-logo { width: 91%; }
+
+  .section-main__wrapper {
+    flex-wrap: wrap;
+    width: auto;
+    justify-content: space-around;
+    margin-bottom: 56px;
+  }
+
+  .character,
+  .section-main__item_info { justify-content: center; }
+  .section-main__item_info,
+  .section-main__item_btns { flex: none; }
+  .section-main__item_info {
+    width: 100%;
+    margin-bottom: 48px;
+  }
+
+  .character {
+    align-items: stretch;
+    flex: .37;
+  }
+  .character,
+  .btn_test-drive { margin-right: 0; }
+
+  .character__item_icon { margin-right: 7px; }
+  .character__item_icon-left { width: 27%; }
+  .character__item_icon-right { width: 20%; }
+  
+  .section-main__item_btns { width: 64%; }
+
+  .button_main {
+    justify-content: space-around;
+    height: 63px;
+  }
+
+  .btn_test-drive { padding-left: 14px; }
+  
+  .btn__item_icon { margin-right: 16px; }
+
+  .button__item_icon-main { width: 30px; }
+
+  .button__item_link { color: #e51937; }
+  
+  .button__item_alt-right {
+    padding-right: 15px;
+    flex: .7;
+  }
+
+  .button__item_big-order {
+    height: 52px;
+    padding-left: 0;
+  }
+
+  .btn__item_order {
+    background-color: #e51937;
+    color: #fff;
+  }
+}
+
+@media (max-width: 510px) {
+  .section-main { background-position: 26% 24%; }
+
+  .btn__item_icon { margin-right: 5px; }
+  
+  .button__item_big { padding-left: 25px; }
+}
+
+/* Телефон */
+@media (max-width: 375px) {
+  .header {
+    height: 47px;
+    margin-bottom: 2px;
+  }
+
+  .header__wrapper_logo {
+    justify-content: flex-start;
+    flex: .4;
+  }
+
+  .header__menu { flex: .175; }
+  .header__link-home { width: 91%; }
+
+  .burger__icon {
+    width: 23px;
+    height: 2.5px;
+    margin-bottom: 4px;
+    border-radius: 0 0 6px 6px;
+  }
+
+  .section-main {
+    margin-bottom: 0;
+    background-position: 40% 28%;
+    background-size: 114%;
+  }
+
+  .section-main__title { height: 81px; }
+
+  .section-main__subtitle { margin-bottom: 46px; }
+
+  .section-main__img { margin-bottom: 22px; }
+
+  .car-logo { width: 96%; }
+  .section-main__wrapper {
+    justify-content: space-around;
+    margin-bottom: 20px;
+  }
+
+  .section-main__item_info {
+    justify-content: space-around;
+    width: 90%;
+    margin-bottom: 30px;
+  }
+
+  .character {
+    height: 48px;
+    justify-content: center;
+    align-items: center;
+    padding-top: 2px;
+    flex: .44;
+  }
+
+  .character__item_icon-left { width: 31%; }
+
+  .character_left { padding-left: 4px; }
+
+  .character_right { padding-right: 7px; }
+
+  .section-main__item_btns { width: 95%; }
+
+  .button__item_icon-main { width: auto; }
+
+  .button__item_alt-right {
+    height: 37px;
+    padding-right: 12px;
+    flex: .74;
+  }
+}
+
+/* Адаптив второй секции */
+
+/* Desktop */
+@media (max-width: 1150px) {
+  .section-desc { height: 780px; }
+
+  .section-desc__wrap_up { padding-top: 0;}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)